frontend/src/routes/habits/[id]/edit/+page.svelte

```diff
@@ -1,38 +1,24 @@
 <script>
-    import { onMount } from "svelte";
+  /** @type {import('./$types').ActionData} */
+  export let form;
+  import Header from "../../../../lib/components/Header.svelte";
+  import FormEars from "../../../../lib/components/FormEars.svelte";
+  import ShadowsForForms from "../../../../lib/components/ShadowsForForms.svelte";
+  import AuthInput from "../../../../lib/components/authInput.svelte";
+  import TimePicker from "../../../../lib/components/TimePicker.svelte";
+  export let data;
 
-    /** @type {import('./$types').ActionData} */
-    export let form;
-    import Header from "../../../../lib/components/Header.svelte";
-    import FormEars from "../../../../lib/components/FormEars.svelte";
-    import ShadowsForForms from "../../../../lib/components/ShadowsForForms.svelte";
-    import AuthInput from "../../../../lib/components/authInput.svelte";
-    export let data;
-
-    onMount(() => {
-        function addNulIfNeeded(event) {
-            if (event.target.value >= 0 && event.target.value <= 9) {
-                event.target.value = `0${event.target.value}`;
-            }
-        }
-
-        const start_time_minutes =
-            document.getElementById("start_time_minutes");
-        const start_time_hours = document.getElementById("start_time_hours");
-        const duration_seconds = document.getElementById("duration_seconds");
-
-        start_time_minutes.addEventListener("change", addNulIfNeeded);
-        start_time_hours.addEventListener("change", addNulIfNeeded);
-        duration_seconds.addEventListener("change", addNulIfNeeded);
-    });
+  // date for time picker
+  let date = new Date();
+  $: _date = date.toLocaleTimeString("en-GB", { timeStyle: 'short' });
 </script>
 
 <div class="h-screen flex justify-center items-center">
     <div class="flex flex-col items-center">
         <FormEars />
         <div class="grid grid-cols-1">
             <Header title="Edit habit" route="habits" displayBackButton="0" />
-            <ShadowsForForms width={307} height={373} />
+            <ShadowsForForms width={267} height={327} />
```

Comment:
Use tailwind to it's maximum potential, don't block it with specific heights and so, but this element I think is for another refactoring